blogsite/myapp/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth import get_user_model
+
+UserModel = get_user_model()
+
+# Create your models here.
+class Topic(models.Model):
+    topic_name = models.CharField(max_length=64, unique=True)
+    description = models.CharField(max_length=255)
+    users = models.ManyToManyField(UserModel, through='UserTopicRelation')
+
+
+class Article(models.Model):
+    title_article = models.CharField(max_length=255, default='Input your text')
+    text = models.TextField(default='Input your text')
+    created_date = models.DateTimeField(auto_now_add=True)
+    update_date = models.DateTimeField(auto_now=True)
+
+    topics = models.ManyToManyField(Topic)
+    author = models.ForeignKey(UserModel, on_delete=models.CASCADE, null=True)
```

Comment:
```suggestion
    author = models.ForeignKey(UserModel, on_delete=models.CASCADE)
```
Не треба `null=True`, у кожної статті повинен бути автор.